core-java-modules/core-java-11-2/pom.xml

```diff
@@ -82,10 +71,10 @@
         <maven.compiler.target.version>11</maven.compiler.target.version>
         <guava.version>29.0-jre</guava.version>
         <mockserver.version>5.11.1</mockserver.version>
-        <jakarta.ws-api.version>3.0.0</jakarta.ws-api.version>
-        <jaxws-rt.version>3.0.0</jaxws-rt.version>
-        <jaxws-ri.version>2.3.1</jaxws-ri.version>
-        <jaxws-maven-plugin.version>2.3.2</jaxws-maven-plugin.version>
+        <jakarta.ws-api.version>3.0.1</jakarta.ws-api.version>
+        <jaxws-rt.version>3.0.2</jaxws-rt.version>
```

Comment:
You deleted the jaxws* dependencies, do we still need the jaxws* properties?

Reply:
Yep removed unused properties. 
This will require changes to https://www.baeldung.com/java-soap-web-service.